### PR TITLE
File status checks failed page

### PR DESCRIPTION
**Pull request by MancunianSam**
I've changed this now so there's only one records result page and it renders success or error depending on the response from the API. This has made it a lot simpler.

### CODE REVIEW

File: test/controllers/FileChecksResultsControllerSpec.scala [hunk 1]
```diff
@@ -1,15 +1,20 @@
 package controllers
 
 import java.util.UUID
-
 import com.github.tomakehurst.wiremock.WireMockServer
-import com.github.tomakehurst.wiremock.client.WireMock.{okJson, post, urlEqualTo}
+import com.github.tomakehurst.wiremock.client.WireMock.{equalToJson, okJson, post, urlEqualTo}
```
Comment (TomJKing) [inline test/controllers/FileChecksResultsControllerSpec.scala:5]: Is the `equalToJson` import needed?

**Review by TomJKing — COMMENTED** [review]
Looks good. One very minor comment. Also did you check this change doesn't break the e2e tests?

File: test/controllers/FileChecksResultsControllerSpec.scala (hunk 1 above)
Comment (MancunianSam) [reply test/controllers/FileChecksResultsControllerSpec.scala]: No, I've removed it.

**Comment by MancunianSam** [comment]
There are changes to the e2e tests on this https://github.com/nationalarchives/tdr-e2e-tests/pull/104 PR which you've already approved. I ran them against this branch and they were ok.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**